osu.Game/Skinning/LegacyJudgementPieceOld.cs

```diff
@@ -55,6 +58,12 @@ public virtual void PlayAnimation()
                     this.ScaleTo(1.6f);
                     this.ScaleTo(1, 100, Easing.In);
 
+                    if (this.version > 1)
```

Comment:
This check cannot exist. There are more than just version checks to decide whether a skin is using the "new" version or not, for instance it will use a newer version if no version is specified (ie. a user putting files into the `User` skin folder with no `skin.ini`).

You should be using something along the lines of:

https://github.com/ppy/osu/blob/e9571be4abac6f16569fb034ef8b64bae9e2381b/osu.Game.Rulesets.Osu/Skinning/Legacy/LegacyMainCirclePiece.cs#L159-L161

(copy paste this code is best since it's the same conditional at stable's end in both usages.

Reply:
I've adjusted this check to match precisely. While `>1` and `>=2.0m` might look the same, let's try and keep the code consistent for our own sanity sake.